Stresser/ProcessArtifact.cpp

```diff
@@ -3,9 +3,10 @@
 ProcessArtifact::ProcessArtifact(const std::string& name, const std::string& data) : IArtifact(name, data)
 {
     std::wstring tempPath = LocalPcUtils::getTempPath();
-    std::wstring wideFileName = std::wstring(ATL::CA2W(name.c_str()));
+    std::wstring wideFileName = std::wstring(ATL::CA2W(data.c_str()));
 
     this->filePath = std::wstring(tempPath + wideFileName);
+    this->pid = 0;
```

Comment:
I thing `0` is also used to represent `System` process in Windows.
Suggestion: Maybe better to use negative sign, like: `-1`.

Reply:
DWORD is unsigned long